### PR TITLE
Add to the halmchart that tenants can create kafka topics

### PR DESCRIPTION
**Pull request by jacobcrawford**
**Description of your changes:**
Add to the helm chart that tenants can create Kafka topics and do some limited configuration

Documentation will come in a later task as it depends on the format in another user-story. 
Checklist:

* [x] I have bumped the chart version
* [ ] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

**Review by jaxels10 — CHANGES_REQUESTED** [review]
Just one comment

**Comment by klausenbusk** [comment]
Please rebase to get https://github.com/distributed-technologies/helm-charts/pull/386, so the CI is green :)

File: charts/tenant-base/chart/values.yaml [hunk 1]
```diff
@@ -71,3 +71,11 @@ secretStore:
...
 #  - name: token
 #    key: secret/tenant-foobar/token
+
+kafkaTopics: []
+#  - name: required
+#    replicas: default 3
+#    partitions: default 1
+#    retention:
+#      ms: default 604800000
+#      bytes: default -1
```
[the remark this answers is not in the document]
Comment (jacobcrawford) [reply charts/tenant-base/chart/values.yaml]: I have added new comments after looking at the Grafana charts

**Review by jaxels10 — APPROVED** [review]
This LGTM

**Pull request merged**